framework/src/auxkernels/ParsedAux.C

```diff
@@ -66,15 +79,31 @@ ParsedAux::ParsedAux(const InputParameters & parameters)
   if (_enable_jit)
     _func_F->JITCompile();
 
+  // obtain current time
+  if (_use_xyzt)
+  {
+    auto fpb = dynamic_cast<FEProblemBase *>(&_subproblem);
+    if (!fpb)
+      paramError("use_xyzt", "Cannot access simulation time t. Set 'use_xyzt' to false.");
+    _time = &fpb->time();
```

Comment:
I need to jump through this little hoop to get access to time. What cases are there that could trigger the error. And would it be better here to just set the time variable to zero and let users carry on?